src/Symfony/Component/HttpKernel/EventListener/ErrorListener.php

```diff
@@ -168,17 +164,38 @@ public static function getSubscribedEvents(): array
     /**
      * Logs an exception.
      */
-    protected function logException(\Throwable $exception, string $message, string $logLevel = null): void
+    protected function logException(\Throwable $exception, string $message, string $logLevel): void
```

Comment:
As this method is **protected**, should we avoid the change in the signature here? 
If yes, we can add another call to the `resolveLogLevel` method inside a null check for the `$logLevel` argument?

Reply:
indeed, the BC break should be prevented